cmd/entrypoint/waiter_test.go

```diff
@@ -122,3 +123,58 @@ func TestRealWaiterWaitWithContent(t *testing.T) {
 		t.Errorf("expected Wait() to have detected a non-zero file size by now")
 	}
 }
+
+func TestRealWaiterWaitWithErrorWaitfile(t *testing.T) {
+	tmp, err := ioutil.TempFile("", "real_waiter_test_file*.err")
```

Comment:
Can we remove the `*` from the filename here, and below?

Reply:
Cannot remove this as, while creating the tempfile we are passing `real_waiter_test_file*.err` as a pattern and not the exact filename. When a file is created with just ioutil.TempFile(`real_waiter_test_file.err`) the resulting name of the file becomes something along the line of `real_waiter_test_file.errcw3987tn4`. If we use `real_waiter_test_file*.err` as the pattern we get `real_waiter_test_filecw3987tn4.err` instead. Having `.err` at the end here is necessary for testing. Hence the `*`.